tests/common.go

```diff
@@ -3494,75 +3504,146 @@ func DeleteSchedule(backupScheduleName string, clusterName string, orgID string,
 	return nil
 }
 
-// CreateSourceAndDestClusters creates source and destination cluster
+// CreateApplicationClusters adds application clusters to backup
 // 1st cluster in KUBECONFIGS ENV var is source cluster while
 // 2nd cluster is destination cluster
-func CreateSourceAndDestClusters(orgID string, cloudName string, uid string, ctx context1.Context) error {
-	// TODO: Add support for adding multiple clusters from
-	// comma separated list of kubeconfig files
+func CreateApplicationClusters(orgID string, cloudName string, uid string, ctx context1.Context) error {
```

Comment:
It is safe to create a new function called `CreateApplicationClusters` and call `CreateSourceAndDestClusters` inside it instead of changing the name of `CreateSourceAndDestClusters`, as changing the name might cause a regression.

Reply:
@mkoppal-px 
As suggested by Mithun, the name have been changed.
Also have made changes in all the test cases using CreateSourceAndDestClusters. 
Once the review is done by all, will start a full run as well

Reply:
Lets change the name once and for all and fix this so that it is future proof

Reply:
Is there anyway to test this regression? I do not think it is sufficient to run our tests.

Reply:
Will run the testcases both with RKE and k8s as scheduler

Reply:
Is our team the only one using the CreateSourceAndDestClusters function? If so, there won't be any issue with changing the function's name.